packages/generator-single-spa/src/root-config/templates/src/root-config.js

```diff
@@ -2,11 +2,18 @@ import { registerApplication, start } from "single-spa";
 import * as isActive from "./activity-functions";
 
 registerApplication({
-  name: "@<%- orgName %>/navbar",
-  app: () => System.import("@<%- orgName %>/navbar"),
-  activeWhen: isActive.navbar,
+  name: "@single-spa/welcome",
+  app: () =>
+    System.import("https://dist-qqbprggud.now.sh/single-spa-welcome.js"),
```

Comment:
TODO: Update with production url once deployed.

Reply:
Could we just use unpkg instead of now.sh? That way a publish to npm would result in a deploy?

```suggestion
    System.import("https://unpkg.com/single-spa-welcome/dist/single-spa-welcome.js"),
```

Reply:
Yep! I'll just need to work with whatever the CI process is to deploy it as its own package. Is that just a lerna command?

Reply:
`yarn publish-all` does this. It calls `lerna publish`, which will publish all the packages in the monorepo at once